### ISSUE TITLE
Wrong traffic size calculation

### ISSUE THREAD

**Issue by AliM1988** [hossinasaadi/x-ui#162]
### Welcome

- [X] Yes, I'm using the latest major release. Only such installations are supported.
- [X] Yes, I'm using the supported system. Only such systems are supported.
- [X] Yes, I have read all WIKI document,nothing can help me in my problem.
- [X] Yes, I've searched similar issues on GitHub and didn't find any.
- [X] Yes, I've included all information below (version, config, log, etc).

### Description of the problem,screencshot would be good

The total of traffic size of clients within a inbound are not equal with traffic size of inbound. Please see the screenshot.
![Screenshot (14)](https://user-images.githubusercontent.com/29524232/206601529-fc8c7147-07f5-40e0-ba7b-a89d1f1dd322.jpg)

### Version of x-ui

<details>

```console
$ x-ui version = 0.5.4
# Paste output here
```

</details>

### x-ui log or xray log

<details>

```console
# paste log here
```

</details>

**Comment by kavkav123** [comment]
Are your used email accounts  in this inbound connection uniq?

**Comment by AliM1988** [comment]
Yes, actually this version doesn't allow duplicated email. Also, these emails had not been used before.
